UnitTests/UnitTests.Shared/Diagnostics/Test_Guard.cs

```diff
@@ -155,26 +155,65 @@ public void Test_Guard_IsNotEqualTo_Fail()
         [TestMethod]
         public void Test_Guard_IsBitwiseEqualTo_Ok()
         {
+            // tests the >16 byte case where the loop is called
+            var biggerThanLimit = new BiggerThanLimit(0, 3, ulong.MaxValue, ulong.MinValue);
```

Comment:
Really last nit about this method - just to improve readability would you mind moving the related temp variables closer together and separating them more clearly? Eg. you're creating that `BiggerThanLimit` value here but only using it a few lines below. Ideally we should have:
- Tests with primitives
- [blank line]
- `Guid` declaration
- `Guid` test
- [blank line]
- `BiggerThanLimit` declaration
- `BiggerThanLimit` test

Reply:
Done!